app/pages/Help.js

```diff
@@ -103,6 +409,7 @@ class Help extends React.Component {
           </li>
         </ul>
 
+        {/* Keyboard Shortcuts */}
```

Comment:
Not a huge fan of these meaningless comments, since you can see the section directly under it anyway. Would suggest to remove these.